Mobile Test Runner for Jama/Mobile Test Runner for Jama/MaxTextLength.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MaxTextLength.swift
+//  Mobile Test Runner for Jama
+//
+//  Created by PSU1 on 7/5/17.
+//  Copyright © 2017 Jaca. All rights reserved.
+//
+
+import Foundation
```

Comment:
I think this file should be moved into the utils file and out of the viewController file. and we should write a test or two to make sure this is working.